src/main/webapp/app/shared/texts/RocReviewLanguage.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+const RocReviewLanguage = () => {
+  return (
+    <div className={'alert alert-warning'}>
+      Only send to MSK ROC review based on either one of the following triggers:
+      <ol style={{ listStyleType: 'lower-alpha' }}>
+        <li>
+          The licensee is located in any of the following countries - Burma,
+          Cambodia, Cuba, China (including Hong Kong and Macau), Iran, North
+          Korea, Russia, Syria, Venezuela, or Belarus; or
```

Comment:
Do we want to keep the `; or` at the end of the first `li`?

Reply:
Yup, this is the legal language to emphasize the second point.